server/graphql/programs/src/queries/patient.rs

```diff
@@ -73,6 +75,7 @@ impl PatientFilterInput {
             country: country.map(StringFilter::from),
             email: email.map(StringFilter::from),
             identifier: identifier.map(StringFilter::from),
+            name_and_code: name_and_code.map(StringFilter::from),
```

Comment:
This would already work without a special filter option. All filters are evaluated as AND so if you specify name and code filters in your request you should get the same result. This is different for OR...

Or do I miss something?

Reply:
Ok so you are doing an or query, right? rename it to name_or_code?

Reply:
Sorry it's name_or_code XD